test/resque_job_test.rb

```diff
@@ -23,7 +23,8 @@
         :configuration => :default,
         :populate_stored_in => nil,
         :_action       => :synchronize,
-        :last_update_ts => nil
+        :last_update_ts => nil,
+        :errors => nil
```

Comment:
This seems like a strange place to need to add the new option 😅 Would it make more sense to add it in the `before` setup of the new `RemoteFiles::File`?

Reply:
I tried adding it there when I first worked on it but it didn't work, I think it's because the  places where I added new options are the expectations of what it's supposed to look like